### ISSUE TITLE
Focus tabbing on a large HTML table causes a long delay until narration begins

### ISSUE THREAD

**Issue by Haditsch** [nvaccess/nvda#13229]
### Steps to reproduce:
1. Click on this codepen link: https://codepen.io/SAH_riag/pen/PoJyXKJ
2. Click down in the display area
3. Tab until table is focused. 
4. Wait for 30 sec. to 1 min until NVDA reads table

### Actual behavior:
NVDA should read the contents of the table without much delay

### Expected behavior:
NVDA is delayed corresponding to the size of the table. 

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
2021.3.1

#### Windows version:
Windows 10 Enterprise 1909

#### Does the issue still occur after restarting your computer?
Yes

**Comment by Brian1Gaff** [comment]
What browser are you using?
 I see no large delay in Waterfox. Of course not knowing much about html 
editors I may not be seeing, if you get my drift, what you are.
 Brian

***@***.***
Sent via blueyonder.
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message ----- 
From: "Haditsch" ***@***.***>
To: "nvaccess/nvda" ***@***.***>
Cc: "Subscribed" ***@***.***>
Sent: Tuesday, January 11, 2022 3:02 PM
Subject: [nvaccess/nvda] Focus tabbing on a large HTML table causes a long 
delay until narration begins (Issue #13229)

**Comment by Haditsch** [comment]
OK, I updated the issue because I checked with a colleague and on his end the delay is not as large, but not instant aswell. At 5.000 lines of code for that table it took his NVDA ~2 sec until narration started, at 17.000 lines 5 seconds. It seems that the time until narration starts scales with the size of the table, but only when tab focusing it, not when pressing T. Then it narrates instantly, also for me where it took ~30 sec before.
